requests/intel-compiler-repack-add-umf.yml

```diff
@@ -0,0 +1,4 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  # this entry adds a package name
+  - intel-compiler-repack: "umf-*"
```

Comment:
Is there a reason you need the entire `umf-*` namespace?

Reply:
@beckermr I am not exactly sure. I was trying to accomplish getting a `true` here:  `"linux-64/umf-2025.0.0-h7a4b287_590.conda": false`.   I thought, to accommodate future versions, I need `umf-*`, but I am happy to stand corrected

Reply:
Ah ok. It is just the package name, so you only need `umf`.